students/Scott_Hancock/session07/run_html_render.py

```diff
@@ -0,0 +1,228 @@
+
+page.append(body)
+
+render_page(page, "test_html_output2.html")
+
+# Step 3
+##########
+
+page = hr.Html()
+
+head = hr.Head()
+head.append(hr.Title("PythonClass = Revision 1087:"))
+
+page.append(head)
+
+body = hr.Body()
+
+body.append(hr.P("Here is a paragraph of text -- there could be more of them, "
+                 "but this is enough  to show that we can do some text"))
+body.append(hr.P("And here is another piece of text -- you should be able to add any number"))
+
+page.append(body)
+
+render_page(page, "test_html_output3.html")
+
+# Step 4
+##########
+
+page = hr.Html()
+
+head = hr.Head()
+head.append(hr.Title("PythonClass = Revision 1087:"))
+
+page.append(head)
+
+body = hr.Body()
+
+body.append(hr.P("Here is a paragraph of text -- there could be more of them, "
+                 "but this is enough  to show that we can do some text",
+              style="text-align: center; font-style: oblique;"))
+
+page.append(body)
+
+render_page(page, "test_html_output4.html")
+
+# Step 5
+#########
+
+page = hr.Html()
+
+head = hr.Head()
+head.append(hr.Title("PythonClass = Revision 1087:"))
+
+page.append(head)
+
+body = hr.Body()
+
+body.append(hr.P("Here is a paragraph of text -- there could be more of them, "
+                 "but this is enough  to show that we can do some text",
+              style="text-align: center; font-style: oblique;"))
+
+body.append(hr.Hr())
+
+page.append(body)
+
+render_page(page, "test_html_output5.html")
+
+# Step 6
+#########
+
+page = hr.Html()
+
+head = hr.Head()
+head.append(hr.Title("PythonClass = Revision 1087:"))
+
+page.append(head)
+
+body = hr.Body()
+
+body.append(hr.P("Here is a paragraph of text -- there could be more of them, "
+                 "but this is enough  to show that we can do some text",
+              style="text-align: center; font-style: oblique;"))
+
+body.append(hr.Hr())
+
+body.append("And this is a ")
+body.append( hr.A("http://google.com", "link") )
+body.append("to google")
+
+page.append(body)
+
+render_page(page, "test_html_output6.html")
+
+# # Step 7
+# #########
+
+page = hr.Html()
+
+head = hr.Head()
+head.append(hr.Title("PythonClass = Revision 1087:"))
+
+page.append(head)
+
+body = hr.Body()
+
+body.append( hr.H(2, "PythonClass - Class 6 example") )
+
+body.append(hr.P("Here is a paragraph of text -- there could be more of them, "
+                 "but this is enough  to show that we can do some text",
+              style="text-align: center; font-style: oblique;"))
+
+body.append(hr.Hr())
+
+list = hr.Ul(id="TheList", style="line-height:200%")
+
+list.append( hr.Li("The first item in a list") )
+list.append( hr.Li("This is the second item", style="color: red") )
+
+item = hr.Li()
+item.append("And this is a ")
+item.append( hr.A("http://google.com", "link") )
+item.append("to google")
+
+list.append(item)
+
+body.append(list)
+
+page.append(body)
+
+render_page(page, "test_html_output7.html")
+
+# # Step 8
+# ########
+
```

Comment:
You can also wrap these into functions that will make it a little cleaner and easier to expand on... or maybe a class of its own ;)